import-service/src/handlers/import-products-file/import-products-file.js

```diff
@@ -0,0 +1,26 @@
+import AWS from 'aws-sdk';
+
+import { response } from '../../utils';
+
+export const importProductsFile = async (event) => {
+  try {
+    const { name } = event.queryStringParameters;
+    const path = `uploaded/${name}`;
+
+    const s3 = new AWS.S3({ region: 'us-east-1' });
+
+    const params = {
+      Bucket: 'import-products',
+      Key: path,
+      Expires: 60,
+      ContentType: 'text/csv',
```

Comment:
Import is working incorrect (I can import whatever)

Reply:
ok, i will fix it after crosscheck, but it is out of scope

Reply:
Asking as seams like I do not think that your s3 settings are correct

Reply:
what confuses you?

Reply:
Your bucket is public and should be not (as I can download back image that uploaded) For me seams like it is not working correctly

Reply:
Summarizing: for me seams like s3 bucket has public access as I can access files via link, and remove is not working as after uploading I can download uploaded file from uploaded folder

Reply:
yes, you are right that I left open public access, now closed it

Reply:
![Screenshot from 2020-11-17 13-03-23](https://user-images.githubusercontent.com/45462253/99375989-64401b00-28d5-11eb-8813-452fee7d0199.png)
![Screenshot from 2020-11-17 13-03-28](https://user-images.githubusercontent.com/45462253/99375997-66a27500-28d5-11eb-8ac1-e29123a1fbb3.png)
![Screenshot from 2020-11-17 13-03-36](https://user-images.githubusercontent.com/45462253/99376005-686c3880-28d5-11eb-9874-7ba2295701be.png)

Reply:
Thank you, now it is working

Reply:
thank you for checking thoroughly